### PR TITLE
Add piped-base image

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
When using pipectl on our CI, we need the `git` command.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: dockers/pipectl-base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.13
```
Comment (khanhtc1202) [inline dockers/pipectl-base/Dockerfile:1]: how about `latest` tag 👀

Comment (nakabonne) [reply dockers/pipectl-base/Dockerfile:1]: I couldn't make up my mind, but as you said, using the latest can reduce our maintenance cost. Besides, seems like alpine isn't likely to make a breaking change.

Comment (nghialv) [reply dockers/pipectl-base/Dockerfile:1]: No, do not use the latest. Keep the tools/dependencies immutable.

Comment (nakabonne) [reply dockers/pipectl-base/Dockerfile:1]: We basically keep our mind that not using the latest, every time we refer to the specific version. But looks like alpine can be an exception. 

@nghialv What do you think?

Comment (nakabonne) [reply dockers/pipectl-base/Dockerfile:1]: Ah, again browser refresh's stuff 😓 
Okay, I'd follow that!

Comment (khanhtc1202) [reply dockers/pipectl-base/Dockerfile:1]: 🙆‍♀️ I do not have strong opinion on that so let's make it as is 👌

**Comment by khanhtc1202** [comment]
🚀 
/approve

**Pull request merged**